TombEngine/Game/control/los.cpp

```diff
@@ -183,32 +183,16 @@ bool GetTargetOnLOS(GameVector* origin, GameVector* target, bool drawTarget, boo
 							else
 							{
 								if (item->ObjectNumber >= ID_SMASH_OBJECT1 && item->ObjectNumber <= ID_SMASH_OBJECT8)
+								{
 									SmashObject(itemNumber);
+								}
 								else
 								{
-									switch (object->hitEffect)
```

Comment:
Why was the case switch for effects removed? Is it not needed?

Reply:
DefaultItemHit do that itself :)